docs/src/modules/components/CodeCopyButton.tsx

```diff
@@ -20,8 +20,8 @@ export default function CodeCopyButton(props: CodeCopyButtonProps) {
       aria-label="Copy the code"
       type="button"
       className="MuiCode-copy"
-      onClick={async (event) => {
-        event.stopPropagation();
+      onClick={async () => {
+        // event.stopPropagation();
```

Comment:
I commented such that if someone investigates a bug related to it they can get  the blam easily